### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by ricardotk**
This series fixes lots of typos in the code

### CODE REVIEW

File: Tools/ArduPPM/Libraries/PPM_Encoder_v3.h [hunk 1]
```diff
@@ -61,7 +61,7 @@ volatile uint8_t servo_input_mode = JUMPER_SELECT_MODE;
 // -------------------------------------------------------------
 
 #define SWITCHOVER_CHANNEL_A		9	// Receiver 1 PPM channel to force receiver 2. Use 0 for no switchover channel
-										// Must be choosed between 6 to 16. Preferabily from 9 to 16 so that APM can use
+										// Must be chose between 6 to 16. Preferabily from 9 to 16 so that APM can use
```
Comment (lgywata) [inline Tools/ArduPPM/Libraries/PPM_Encoder_v3.h]: I think it's "Must be chosen"

**Comment by lgywata** [comment]
Only one comment. Other than that, LGTM.

**Comment by lucasdemarchi** [comment]
[codespell](https://github.com/lucasdemarchi/codespell) use detected :)

**Comment by rmackay9** [comment]
very nice.

**Comment by lucasdemarchi** [comment]
I think this will cause lots of conflicts for pending PRs. So maybe we should get in the patches that don't touch recently changed parts of the codebase and then figure out the right moment to get the rest in. @rmackay9 what do you think?

**Comment by OXINARF** [comment]
@lucasdemarchi Why would it cause conflicts? This is just changing comments, I don't think there are many PRs changing comments.

**Comment by lucasdemarchi** [comment]
@OXINARF because they could be changing code nearby the comments. It's easy to solve the conflict while rebasing, but it will depend on what's the experience with git the people submitting the PR have.

**Comment by OXINARF** [comment]
@lucasdemarchi Ok. I wouldn't delay these commits because of those (small?) cases but that's your call.

**Comment by lucasdemarchi** [comment]
@OXINARF ok. You convinced me :)

Applied.

I just removed a change to  Tools/SerialProxy/SerialProxy/Form1.cs since I don't know if we have to escape `'` inside a string there and I prefer not to break it.

**Pull request closed without merging**